src/test/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraperTest.java

```diff
@@ -192,6 +193,21 @@ public void testWriteTagCheck() throws Exception {
                 "expected correct parse");
     }
 
+    @Test
+    public void testViolationsEmptyDescriptionModule() throws Exception {
```

Comment:
The existing `MetadataGeneratorUtilTest#testMetadataFilesGenerationAllFiles` needs to be expanded to print the violations and check what they are.

Reply:
To be done in a separate PR-
> The existing MetadataGeneratorUtilTest#testMetadataFilesGenerationAllFiles needs to be expanded to print the violations 

It will print the violations.

>...and check what they are.

Again, you want to check the number of violations or the exact violation.

Reply:
Will create a separate PR for this after current gets merged.

Reply:
It should be connected to the change with the util. It will show your changes are working.

Reply:
I see all the changes to the util are removed now.